ballerina/client.bal

```diff
@@ -256,6 +256,70 @@ public isolated client class Client {
         http:Response httpResponse = check self.amazonS3->delete(requestURI, request);
         return handleHttpResponse(httpResponse);
     }
+    
+    # Generates a presigned URL for the object.
+    #
+    # + bucketName - The name of the bucket  
+    # + objectName - The name of the object  
+    # + httpMethod - The HTTP method to be used, either GET or PUT  
+    # + expires - The time period for which the presigned URL is valid, in seconds  
+    # + partNo - The part number of the object, when uploading multipart objects  
+    # + uploadId - The upload ID of the multipart upload  
+    # + objectCreationHeaders - Optional headers for GET http method 
```

Comment:
```suggestion
    # + objectCreationHeaders - Optional headers for GET HTTP method 
```

Reply:
Are these correct anyway? Isn't `objectCreationHeaders` for PUT?

Reply:
yes